query.go

```diff
@@ -749,14 +749,8 @@ func (s *ChainService) GetCFilter(blockHash chainhash.Hash,
 		return nil, fmt.Errorf("unknown filter type: %v", filterType)
 	}
 
-	// Only get one CFilter at a time to avoid redundancy from mutliple
```

Comment:
Without this mutex, we'll have multiple concurrent requests for the same set of filters.